core/src/main/java/jenkins/ClassLoaderReflectionToolkit.java

```diff
@@ -70,6 +70,7 @@ private static Object getClassLoadingLock(ClassLoader cl, String name) {
     /**
      * Calls {@link ClassLoader#findLoadedClass} while holding {@link ClassLoader#getClassLoadingLock}.
      * @since 1.553
+     * @deprecated use {@link #loadClass(ClassLoader, String)}
```

Comment:
is there a reason we need to deprecate rather than remove?

Reply:
I suppose not. Then again, I am just following the precedent set in commit 898f1f76a3.